pkg/config/core/resources/apis/config.go

```diff
@@ -1,7 +1,33 @@
 package apis
 
+import (
+	"time"
+
+	config_types "github.com/kumahq/kuma/pkg/config/types"
+)
+
 type Config struct {
+	// List of enabled core resources
 	Enabled []string `json:"enabled" envconfig:"KUMA_CORE_RESOURCES_ENABLED" default:""`
+	// Status of core resources
+	Status ConfigStatus `json:"status"`
+}
+
+type ConfigStatus struct {
+	// How often we compute status of MeshMultiZoneService
+	MeshMultiZoneServiceInterval config_types.Duration `json:"meshMultiZoneServiceInterval" envconfig:"CORE_RESOURCES_STATUS_MESH_MULTI_ZONE_SERVICE_INTERVAL"`
```

Comment:
shouldn't `envconfig` start with `KUMA_CORE_...` as `KUMA_CORE_RESOURCES_ENABLED`?

Reply:
uhhh, it definitely should! thx